archetype/src/main/resources/archetype-resources/Dockerfile

```diff
@@ -89,14 +89,14 @@
 ## Once Open Liberty has a proper release for EE 11, this should be removed.
 #if (${jakartaVersion} == '11')
 #set ($baseImage = "icr.io/appcafe/open-liberty:beta")
+#elseif (${javaVersion} == '21')
+#set ($baseImage = "icr.io/appcafe/open-liberty:full-java21-openj9-ubi-minimal")
```

Comment:
Why is this set to minimal when the other ones are not?

Reply:
This looks correct.  We don't release every combination of Java version and UBI base but only select combinations.

Some references for anyone looking to understand more:
1. From the [tags doc](https://github.com/OpenLiberty/ci.docker/blob/main/docs/icr-images.md) at the Open Liberty Docker source repo  

> Liberty images with Java 21 are based on UBI 9 minimal and include IBM Semeru Runtimes for Java 21 JRE. This combination offers a compact and effective Java runtime that is suited for applications that need Java 21.

> Liberty images with Java 8, 11 and 17 and with the openj9 type are based on UBI 8 standard and include IBM Semeru Runtime for the respective Java version with the JDK. Images with the ibmjava type are based on UBI 8 standard and include IBM Java 8 JRE.

2. Also note to list the available tags in ICR you can do (login required to list, not to pull though):
```
ibmcloud cr region-set global 
ibmcloud cr images --restrict appcafe/open-liberty
```

Reply:
Thanks for reviewing this, @scottkurz. Will the minimal images still work with with full and web profile for EE 10 or lower? Anyway, I’ll double check the docs and tag listing.

Reply:
> Will the minimal images still work with with full and web profile for EE 10 or lower?

Yes that should work.  UBI minimal has fewer utilities and packages installed than UBI standard but can still run a Liberty app.

> should we consider using slim and calling configure.sh?

We could achieve a smaller image size for the end user by starting from the kernel-slim image.   It's actually the `features.sh` script you'd want to run to install the necessary features.    The `configure.sh` script would mainly build a class cache during image build to improve image startup time.      

This all adds to the image build time, though, so it depends whether we prioritize a simpler, quicker build or a better image. 
 
I haven't been too close to the project lately so I'll just mention that without offering a strong opinion.   You can see using the Open Liberty starter:  https://openliberty.io/start/  our Dockerfile there includes does use kernel-slim and invokes both.  

------------------------------------------------------------------------------

BTW I haven't had a chance to build or test any of this recent work myself, I just happened to notice the GitHub issue so was just making a quick comment.   Will try to look more but not sure exactly when I will get back to it.

Reply:
OK. I’d say let’s stick with our current approach then. Our purpose is to have a friendly starting point. Longer image build times may get in the way of that.

Reply:
As @scottkurz said: Not all ubi/ubi-minimal combinations are available:
- full-java21-openj9-ubi-minimal exists
- full-java21-openj9-ubi does not exist
- full-java17-openj9-ubi-minimal does not exist
- full-java17-openj9-ubi exists
Looks like Liberty switch from -ubi to -ubi-minimal for Java 21